bliss/sle/cltu.py

```diff
@@ -327,7 +327,29 @@ def decode(self, message):
     def _bind_return_handler(self, pdu):
         ''''''
         result = pdu['cltuBindReturn']['result']
+        responder_identifier = pdu['cltuBindReturn']['responderIdentifier']
```

Comment:
If this is the same across the 3 interfaces let's move the bulk of the logic to the common class between the 3. I realize there was probably a lot of duplication before as well though =)